src/views/Sale/components/PurchaseTokensForm/index.tsx

```diff
@@ -148,6 +148,8 @@ export const PurchaseTokensForm = ({ saleId }: PurchaseTokensFormComponentProps)
         )} ${sale?.tokenOut.symbol}`
       )
     }
+    console.log({ purchaseMaximumAllocation })
+    console.log({ quantity })
```

Comment:
I think we should remove the logs here, if these were not left intentionally?

Reply:
Thanks, I actually removed them but got distracted before committing the changes.
Removed now

Reply:
Great, everything else looks great.